apisix/plugins/sls-logger.lua

```diff
@@ -126,17 +127,13 @@ local function remove_stale_objects(premature)
 end
 
 local function combine_syslog(entries)
-    local data
+    local data = {}
     for _, entry in ipairs(entries) do
-        if not data then
-           data = entry.data
-        end
-
-        data = data .. entry.data
+        table.insert(data, entry.data)
         core.log.info("buffered logs:", entry.data)
     end
 
-    return data
+    return table.concat(data)
 end
```

Comment:
Better to put `_M.combine_syslog = combine_syslog` here.

Reply:
fixed.